jit.en.vtt

```diff
@@ -735,7 +735,7 @@ against some practical goals, right?
 I don't think anyone wants me to
 
 09:24.640 --> 09:27.720 align:start position:0%
-dump a 10_000 line C++ JIT on us,
+dump a 10,000 line C++ JIT on us,
```

Comment:
I prefer using the Python style, which is clearly not a decimal comma.
I can change it if *Brandt* feels strongly about that.

Reply:
Yeah, I can see the reason to avoid confusion with a decimal comma used in some European countries, but in English, it's `.` for decimal, and `,` for thousands. 

Although there's also the (non-breaking) space sometimes used in scientific texts, or just avoid the separator altogether. Underscores looked a bit odd in the context of subtitles.

Of the tech style guides I checked, [Canonical](https://docs.ubuntu.com/styleguide/en/#numbers), [Microsoft](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/units-of-measure-terms) and [Google](https://developers.google.com/style/numbers#commas-and-decimal-points-in-numbers) say to use commas for numbers with four digits or more; [Red Hat](https://stylepedia.net/style/5.0/#numbers) says don't use commas for numbers with four digits, but use commas for numbers with five digits.

Anyway! cc @brandtbucher to decide :)

Reply:
I mean, if it's up to me, I find the underscore a tiny bit jarring when re-watching that part of the video with subtitles on. I'd prefer the comma.

Reply:
Yup, it's your talk :)